test/v4/helper.js

```diff
@@ -202,14 +199,14 @@ module.exports.calcDstQty = function(srcQty, srcDecimals, dstDecimals, rate) {
     return result;
 }
 
-module.exports.calcRateFromQty = function(srcAmount, dstAmount, srcDecimals, dstDecimals) {
-
+module.exports.calcRateFromQty = function(srcQty, dstQty, srcDecimals, dstDecimals) {
+    let decimals;
     if (dstDecimals >= srcDecimals) {
-            let decimals = new BN(10).pow(new BN(dstDecimals - srcDecimals));
-            return ((precisionUnits.mul(dstAmount)).div(decimals.mul(srcAmount)));
+        decimals = new BN(10).pow(new BN(dstDecimals - srcDecimals));
```

Comment:
why not use let.
in js we use let or const.

Reply:
`decimals` is earlier declared in line 203. This is so that there is only one initialization of `decimals` in the function.

Reply:
Oh
Sorry
missed that.
OK